department-of-veterans-affairs/va.gov-cms
# VACMS-0000 AP work

## Description
hello!

this PR relates to next-build functionality -- the CMS's JSONAPI endpoint wasn't returning all the expected data without these two patches included. Also needed to remove the config for news story node's path value... something apparently related to base field override or translations seems to mysteriously drop some fields in the json response. Removing that config shows the path variable in the response as expected.



## Testing done
running locally, hitting a jsonapi endpoint for a news story node should include a `path` object underneath the `attributes` key:
https://va-gov-cms.ddev.site/jsonapi/node/news_story/1dbd5b1e-0fb2-43fb-8581-a2f9edfe3c83

## Screenshots
![Screenshot 2023-07-13 at 1 36 34 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/c81e4f91-afbb-4a50-9793-23ed00380187)


## QA steps
- What needs to be checked to prove this works?  
Visit the same endpoint without the patches applied and see that the path variable is missing
You could also try running next-build repo as-is, story listing pages will fail because path is not found on any of the listed news story nodes. There are other cleanup tasks to be pushed and merged there as soon as I get VA access to do so :)

- What needs to be checked to prove it didn't break any related things?  
do news stories still translate as expected?

### Definition of Done


### Select Team for PR review


Is an announcement needed to let editors know of this change?
